src/CompanyService.Models.Db/DbDepartment.cs

```diff
@@ -14,6 +14,10 @@ public class DbDepartment
         public string Name { get; set; }
         public string Description { get; set; }
         public bool IsActive { get; set; }
+        public Guid CreatedBy { get; set; }
+        public DateTime CreatedAtUtc { get; set; }
```

Comment:
I think, not need use UTC suffix, so as utc use default everywhere

Reply:
you are not right